src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryPaths.java

```diff
@@ -6,11 +6,10 @@ private MercuryPaths() {}
 
   // Paths in Mercury Curated Main Pages
   public static final String ROOT_PATH = "/";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)